.jenkins/Jenkinsfile-pr-s390x

```diff
@@ -0,0 +1,234 @@
+                    println("EXCLUDE_GROUPS: ${env.EXCLUDE_GROUPS}")
+                    if (env.ghprbCommentBody.contains('test_only')) {
+                        env.TEST_ONLY = "true"
+                        env.DOCKER_REGISTRY = "quay.io"
+                        env.DOCKER_ORG="strimzi"
+                        env.DOCKER_TAG = "latest"
+                    } else {
+                        env.TEST_ONLY = "false"
+                        env.DOCKER_ORG="strimzi"
+                        env.DOCKER_REGISTRY = "localhost:5000"
+                        env.DOCKER_TAG="pr"
+                    }
+                    if (env.ghprbCommentBody.contains('install_type=')) {
+                        env.CLUSTER_OPERATOR_INSTALL_TYPE = env.ghprbCommentBody.split('install_type=')[1].split(/\s/)[0]
+                    }
+                    if (env.ghprbCommentBody.contains('env=')) {
+                        env.ENV_VARS = env.ghprbCommentBody.split('env=')[1]
+                        for (envVar in "${env.ENV_VARS}".split(",")) {
+                            env.(envVar.split("=")[0]) = envVar.split("=")[1]
+                            println("${env.(envVar.split("=")[0])}")
+                        }
+                    }
+                    if (env.ghprbCommentBody.contains('kube_version=')) {
+                        env.KUBE_VERSION = env.ghprbCommentBody.split('kube_version=')[1].split(/\s/)[0]
+                    } else {
+                        // Default version (the oldest supported by Strimzi)
+                        env.KUBE_VERSION = "stable"
+                    }
+
+                    if (env.ghprbCommentBody.contains('test_count_running_in_parallel=')) {
+                        env.TEST_COUNT_RUNNING_IN_PARALLEL = env.ghprbCommentBody.split('test_count_running_in_parallel=')[1].split(/\s/)[0]
+                    } else {
+                        env.TEST_COUNT_RUNNING_IN_PARALLEL = "1"
+                    }
+
+                    println("ENV_VARS: ${env.ENV_VARS}")
+                    env.BUILD_PROJECT_STATUS = false
+                    env.BUILD_DOCKER_IMAGE_STATUS = false
+                    println("TEST_ONLY: ${env.TEST_ONLY}")
+                    println("DOCKER_REGISTRY: ${env.DOCKER_REGISTRY}")
+                    println("DOCKER_ORG: ${env.DOCKER_ORG}")
+                    println("DOCKER_TAG: ${env.DOCKER_TAG}")
+                    println("CLUSTER_OPERATOR_INSTALL_TYPE: ${env.CLUSTER_OPERATOR_INSTALL_TYPE}")
+                    println("KUBE_VERSION: ${env.KUBE_VERSION}")
+                    println("TEST_COUNT_RUNNING_IN_PARALLEL: ${env.TEST_COUNT_RUNNING_IN_PARALLEL}")
+                }
+            }
+        }
+        stage('Start Minikube') {
+            steps {
+                timeout(time: 10, unit: 'MINUTES') {
+                    script {
+                        sh(script: """
+                            sudo dockerd -G ${ST_USER} &
+                            sleep 10
```

Comment:
Sure, I've added a comment in the pipeline script.

We're using a separate docker daemon inside the container so that local registries wouldn't affect each other. 

`docker` daemon is started with `-G` option to ensure the Jenkins user can connect to `/var/run/docker.sock`. Besides `-G`, I also tried other options such as adding jenkins user to group `docker`, but looks like it doesn't work in Jenkins pipeline.